tests/test_wrappers.py

```diff
@@ -1,7 +1,8 @@
 """Tests for wrapper classes."""
+import pytest
 
 from benchmark_environments.testing.envs import CountingEnv
-from benchmark_environments.util import AutoResetWrapper
+from benchmark_environments.util import AutoResetWrapper, EpisodeEndRewardWrapper
```

Comment:
Reiterating https://google.github.io/styleguide/pyguide.html#22-imports comment (although in this file see we're already importing classes)

Reply:
Fixed in both cases